sympy/functions/combinatorial/factorials.py

```diff
@@ -1057,3 +1057,25 @@ def _eval_is_nonnegative(self):
                 return True
             elif k.is_even is False:
                 return  False
+class multinomial(CombinatorialFunction):
+    @classmethod
+    def eval(cls, *k):
+        from sympy.core import Add,Mul
+        k = [sympify(x) for x in k]
```

Comment:
Please these imports at the top of the file.

Reply:
if you take a look at the code for binomial ... this occurrence is fairly common ... I plan to import all of them at the top ... 😄

Reply:
The imports should go at top if they aren't cyclic in nature.

Reply:
I've made changes, but I haven't commited, the next commit might be a big one, I'm writing tests ... so